scripts/test/lib/cluster.sh

```diff
@@ -32,5 +36,5 @@ function set_env_aws_node() {
 
   echo "Setting environment variable $KEY to $VAL on aws-node"
   kubectl set env daemonset aws-node -n kube-system $KEY=$VAL
-  kubectl rollout status ds -n kube-system aws-node
+  check_ds_rollout "aws-node" "kube-system"
```

Comment:
Looks like this takes care of the timeout.

Reply:
Yes, check_ds_rollout() has a default timeout of 2m, which should be good: https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/master/scripts/test/lib/k8s.sh#L36